openapi.yaml

```diff
@@ -95,6 +95,44 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /diff:
+    post:
+      summary: Compare the given AsyncAPI documents.
+      operationId: diff
+      tags:
+        - diff
+      requestBody:
+        description: Compare the given AsyncAPI documents.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DiffDocument'
+      responses:
+        '200':
+          description: The given AsyncAPI documents are equal.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DiffResult'
+        '400':
+          description: The given AsyncAPI documents are not equal.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '422':
+          description: The given AsyncAPI documents are not equal due to invalid parameters in the request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem'
```

Comment:
I don't think so that `200` status should be only for equal documents. `diff` library returns the list of changes between two AsyncAPI documents so `200` should be also returned when diff checks also changes, please change description for `200`. Also `400` and `422` should be returned when AsyncAPI document(s) is/are invalid like in the `/validate` path. You can check implementation for it :)

Reply:
I didn't get what should I write as a description for `200`